tests/component/sections/shared/pagination/PaginationResultsInfo.spec.tsx

```diff
@@ -53,4 +53,17 @@ describe('PaginationResultsInfo', () => {
     )
     cy.findByText('10 of 15 Items seen').should('exist')
   })
-})
+
+  it('shows the correct formatted results when accumulated prop passed and results are more than page size', () => {
+    const totalCount = 1500
+    const expectedFormattedTotalCount = '1,500'
```

Comment:
What do you think that instead of hardcoding the expected result, we use Intl here too?
So we are not hardcoding the expected to any specific language.
`const expectedFormattedTotalCount = new Intl.NumberFormat().format(totalCount)`

Reply:
The two approaches work. 

If it is hard-coded, it is easier to see what is the expected result for that specific language. For en, we know for sure if it "1,500" and we know there is a need for a comma.

If it is not hard-coded, it is slightly easier to reconfigure and run.

I made the change already and I can change it again to whichever is most preferred.

Reply:
Both valid points, thanks for your contribution!